control-plane-operator/controllers/hostedcontrolplane/cluster_policy/config.go

```diff
@@ -0,0 +1,70 @@
+package cluster_policy
```

Comment:
Nit: conventional package name would be `clusterpolicy`

Reply:
thx, updated